services/motion/builtin/builtin_test.go

```diff
@@ -505,6 +505,50 @@ func TestMoveOnMapSubsequent(t *testing.T) {
 	test.That(t, spatialmath.PoseAlmostEqualEps(goalPose2, spatialmath.PoseBetween(goal1BaseFrame, goal2BaseFrame), 10), test.ShouldBeTrue)
 }
 
+func TestMoveOnMapAskewIMU(t *testing.T) {
```

Comment:
1. Can we please move this test case (the success case) into a t.Run?
2. Can we please have failure cases for situations where the skew is too great (i.e. the rover is upside down, straight up & straight down)?

Reply:
Moved into t.Run(). Since the other test introduced here comprehensively tests all errors, I added just one test here confirming that an error created in the new function indeed passes through and causes MoveOnMap to fail.